README.md

```diff
@@ -3,18 +3,16 @@ _Postman is an API platform for building and using APIs. Postman simplifies each
 
 # Postman App Support
 
-If you have a feature request, a new integration idea or you would like to file a bug report, please use this [Github issue tracker](https://github.com/postmanlabs/postman-app-support/issues). 
+If you have a feature request, a new integration idea, or you would like to file a bug report, please use this [Github issue tracker](https://github.com/postmanlabs/postman-app-support/issues). 
 
-We _recommend_ that you search the issue tracker to check if someone else has already reported the issue and whether there is a known solution that you can use. This would be the fastest way for you to find a solution to any issue that you are currently facing.
+We _strongly encourage_ that you search the issue tracker to check if someone else has already reported the issue and whether there is a known solution that you can use. This would be the fastest way for you to find a solution to any issue that you are currently facing.
 
 If you are adding a bug report, please add detailed steps to reproduce the bug, the Postman version you're using, and your OS version. Any additional files (collections, data dumps, console errors, screenshots) would be very helpful and would help us to narrow down the issue as quickly as possible. 
 
-We have compiled a quick set of [guidelines for reporting issues](#guidelines-for-reporting-issues).
```

Comment:
This was a broken link.

Reply:
This should be pointing here - https://github.com/postmanlabs/postman-app-support/blob/support/CONTRIBUTING.md

Reply:
Updated!